example/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-add_executable(inexor-vulkan-renderer-example main.cpp)
+add_executable(inexor-vulkan-renderer-example
```

Comment:
This is not needed just add the rc file into our normal source files. If it is not useable (e.g. linux) that file is ignored.

Reply:
Oh wait mb XD haha miss read.